tests/utilities/test_xla_device_utils.py

```diff
@@ -0,0 +1,31 @@
+import pytest
+
+from pytorch_lightning.utilities.xla_device_utils import XLADeviceUtils as xdu
+from tests.base.develop_utils import pl_multi_process_test
+
+try:
+    import torch_xla.core.xla_model as xm
+    XLA_AVAILABLE = True
+except ImportError as e:
```

Comment:
use `TPU_AVAILABLE` here too instead of `XLA_AVAILABLE`?

Reply:
The purpose of the tests will be missed if I do that. The `XLA_AVAILABLE` variable makes sure that `torch_xla` module is present on the system. The tests determine if the XLA device is a TPU.